src/stubs/ErrorUtils.js

```diff
@@ -11,13 +11,17 @@
 
 /* jslint unused:false */
 
-var ErrorUtils = {
-  applyWithGuard(callback, context, args, onError, name) {
-    return callback.apply(context, args);
-  },
-  guard(callback, name) {
-    return callback;
-  },
-};
+if (global.ErrorUtils) {
```

Comment:
@zpao Confirmed that this has not mapped from an internal file in quite a while, so we're good. :)